airbyte-integrations/connectors/source-pokeapi/source_pokeapi/source.py

```diff
@@ -0,0 +1,84 @@
+
+# Basic full refresh stream
+class PokeapiStream(HttpStream):
+    url_base = 'https://pokeapi.co/api/v2/'
+
+    def __init__(self, name: str, **kwargs):
+        super().__init__()
```

Comment:
The best practice is to pass `**kwargs` to the superclass
```suggestion
        super().__init__(**kwargs)
```